### ISSUE TITLE
Bird Mania 3D

### ISSUE THREAD

**Issue by drewbyhk** [Panda3DS-emu/Panda3DS-Games-List#128]
### Game Name (and version if necessary)

Bird Mania 3D (USA)

### Hardware Specs (If necessary)

- CPU: Ryzen 5 3600X
- GPU: Nvidia RTX 2060

### Operating System

Windows 11 22H2

### Game State

Anytime the game is saved, it will display a message stating: "Could not read save data. Turn the system off and"

This includes loading the game for the first time, creating a name for the first time, dying in the game, etc.

### Screenshots

![image](https://github.com/Panda3DS-emu/Panda3DS-Games-List/assets/5677214/383f10e0-337f-4090-9b8a-b476b7c960b9)
![image](https://github.com/Panda3DS-emu/Panda3DS-Games-List/assets/5677214/d60bc83b-0330-4440-a5e0-870e2e1c57aa)
![image](https://github.com/Panda3DS-emu/Panda3DS-Games-List/assets/5677214/9013c530-b914-4116-b25a-fdaa2cf8f972)

### Additional information

Console states "[Warning] Ignoring mprotect! Hope nothing goes wrong but if the game accesses invalid memory or crashes then we prolly need to implement this" 3 times on startup.

**Comment by drewbyhk** [comment]
I'm doing God's work reporting this game's compatibility. What would the world do if they didn't know Bird Mania 3D is broken?!?!

**Comment by wheremyfoodat** [comment]
This might be fixed by the ExtSaveData PR, will have to check

**Comment by drewbyhk** [comment]
Also forgot to mention, the bird is impossible to control with the touch screen, but works with the Circle Pad/D-Pad.

**Comment by wheremyfoodat** [comment]
Are you using a build from the website? The ones in the latest Github release are outdated.

**Comment by wheremyfoodat** [comment]
https://panda3ds.com/download.html

**Comment by drewbyhk** [comment]
> Are you using a build from the website? The ones in the latest Github release are outdated.

Yes, I just downloaded it from the website this morning. It should be the latest nightly.

**Comment by drewbyhk** [comment]
Just deleted my save and tried loading the game again. The console gave me this: "[Warning] ControlMemory: unknown operation 5" along with the mprotect warnings. Not sure if that helps any.
